### PR TITLE
Make Button component

### PR DESCRIPTION
**Pull request by sophiekrag**

### CODE REVIEW

**Comment by michaelkoek** [comment]
You have this PR twice? Same code is inside the other PR

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
-function App() {
+import React from "react";
+
+
+const App = () => {
   return (
     <>
-    Hello there
+      <p>Hello</p>
```
Comment (guido-an) [inline src/App.js:7]: Try to have in the pull request only the code related to it

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const Button = ({ onClick, children, type = "button" }) => {
+  return (
+    <MainButton onClick={onClick} type={type}>
+      {children}
+    </MainButton>
+  );
+};
+
+const MainButton = styled.button`
+  color: ${(props) => (props.secondary ? "gray" : "white")};
```
Comment (guido-an) [inline src/components/Button.js:13]: You could also deconstruct the object, same in the next lines. This is ok though :)

Comment (michaelkoek) [reply src/components/Button.js:13]: Like said in the other PR; I don't see the `<MainButton>` component taking any prop named `secondary`.
Also; best to store the colors in a theme object and expose it through the app with the `<ThemeProvider>`

**Comment by sophiekrag** [comment]
> You have to PR twice? Same code is inside the other PR

Yes stupid mistake I made...

**Comment by sophiekrag** [comment]
Using just some basic styling right now, will come back to it once I figured out how the look of the app will be

**Pull request closed without merging**